upup/pkg/fi/cloudup/alitasks/securitygrouprule.go

```diff
@@ -49,7 +49,7 @@ func (s *SecurityGroupRule) CompareWithID() *string {
 }
 
 func equalsIgnoreCase(a, b string) bool {
-	if strings.ToLower(a) == strings.ToLower(b) {
+	if strings.EqualFold(a, b) {
```

Comment:
Seems to me that `equalsIgnoreCase`  just reimplements `strings.EqualFold` and is used only once.
Maybe would be best to just remove it and replace where it's used..

Reply:
Sure.  I can do it